<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="lbl_loadTag">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:blue;</string>
    </property>
    <property name="text">
     <string>Carga</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_torqueTag">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:blue;</string>
    </property>
    <property name="text">
     <string>Torque</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_frequencyTag">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:blue;</string>
    </property>
    <property name="text">
     <string>RPM</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_timeTag">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:blue;</string>
    </property>
    <property name="text">
     <string>Tempo</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_currentProtocol">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Manual</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_frequencyValue">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid blue;
border-radius:8;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>14</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_loadValue">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid blue;
border-radius:8;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>14</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_timeValue">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid blue;
border-radius:8;</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>14</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_torqueValue">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>240</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid blue;
border-radius:8;</string>
    </property>
    <property name="text">
     <string>0.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>14</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_loadUnit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Watts</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_torqueUnit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>300</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string> kgf.m</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_inbramed">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/img/logo_inbramed_small.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_play">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>121</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #ff00ff;
border-radius:8;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/img/img/button_play_big.png</normaloff>:/img/img/button_play_big.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_pause">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>340</y>
      <width>121</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #ff00ff;
border-radius:8;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/img/img/button_pause_orange.png</normaloff>:/img/img/button_pause_orange.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_stop">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>121</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #ff00ff;
border-radius:8;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/img/img/button_stop_green.png</normaloff>:/img/img/button_stop_green.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_increaseLoad">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #ff0000;
border-radius:8;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/img/img/button_plus.png</normaloff>:/img/img/button_plus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_decreaseLoad">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #ff0000;
border-radius:8;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/img/img/button_minus.png</normaloff>:/img/img/button_minus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_protocol">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>340</y>
      <width>121</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #ff00ff;
border-radius:8;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/img/img/icon_stopwatch.png</normaloff>:/img/img/icon_stopwatch.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_manual">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Manual</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_timesFive">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>x5</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
